### DISCUSSION TITLE
Crediting

### DISCUSSION THREAD

**Discussion by ChristopherCampDP** [jamland/samplescope#136] [Ideas]
Awesome app. My team does a ton of sourcing of creative commons audio files during their day-to-day. In order for us to fully use this app, we'd need a link back to the source, so we could credit accordingly. Maybe something like a "Copy Credit" button that would copy the title, author, and link to the clipboard. Could even be HTML.

**Comment by jamland** [comment]
Hi @ChristopherCampDP!

Thank you for the feedback. Yes, looks like I overlooked these license details initially.
And I will think how to add it in a simple and useful way.
I will try to implement it with the next update release.
